VideoLocker/src/main/java/org/edx/mobile/module/serverapi/parser/IParser.java

```diff
@@ -0,0 +1,11 @@
+package org.edx.mobile.module.serverapi.parser;
+
+import java.io.Reader;
+import java.util.List;
+
+public interface IParser {
```

Comment:
Is there a case where we'd want to parse something other than JSON with this interface?